posts/2021-07-26-GATs-stabilization-push.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: "The push for GATs stabilization"
+author: Jack Huey
+team: the Traits Working Group <https://www.rust-lang.org/governance/teams/compiler#wg-traits>
+---
+
+# The push for GATs stabilization
+
+Where to start, where to start...
+
+Let's begin by saying: this is a very exciting post. Some people reading this will be overwhelmingly thrilled; some will have no idea what GATs (generic associated types) are (that'll be explained a bit later if this is you); others might be in disbelief. The [RFC] for this feature did get opened in April of *2016* (and merged about a year and a half later). In fact, this RFC even predates const generics. Don't let this fool you though: it is a powerful feature; and the reactions to the tracking issue on Github should maybe give you an idea of its popularity (it is *the* most upvoted issue on the Rust repository):
+![GATs reactions](https://i.imgur.com/6uOZbXj.png)
```

Comment:
Can we put this in `static/images` instead please? Would be nice to have this image forever, rather than losing it if imgur goes away.